.travis.yml

```diff
@@ -83,11 +82,11 @@ script:
 - cd scripts
 - "./run.shellcheck"
 - cd ..
+- export CLAW_BUILD=$(CLAW_SRC)/build
+- export OMNI_HOME=$(CLAW_BUILD)/home
 - if [[ "${TRAVIS_PULL_REQUEST_SLUG}" == "claw-project/claw-compiler" ]]; then sonar-scanner -Dsonar.login=$SONAR_TOKEN; fi
 - if [[ "${TRAVIS_REPO_SLUG}" == "claw-project/claw-compiler" ]] && [[ "${TRAVIS_PULL_REQUEST_SLUG}" == "" ]]; then sonar-scanner -Dsonar.login=$SONAR_TOKEN; fi
 notifications:
-  slack:
```

Comment:
Why stopping to notify on the slack channel?

Reply:
I think we need it to execute some new command

Reply:
I think we are not watching it, but if you do we can keep it.